envoy/admin/admin.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package envoy.admin;
+
+import "google/protobuf/any.proto";
+import "envoy/api/v2/rds.proto";
+
+import "gogoproto/gogo.proto";
+
+message ConfigDump {
+  map<string, google.protobuf.Any> configs = 1 [(gogoproto.nullable) = false];
```

Comment:
Can you add comments? It's hard to know what this very general type is representing.

Reply:
+1. In general, we should add full comments here for which admin endpoints these map to, how they are used, etc. It's not completely clear to me whether we want strongly typed output (maybe we do) vs. something general like this type.